### PR TITLE
fix: update version check login and other minor UI fixes

### PR DESCRIPTION
**Pull request by YounixM**

### CODE REVIEW

**Review by palashgdev — APPROVED** [review]

File: frontend/src/container/BillingContainer/BillingContainer.tsx [hunk 1]
```diff
@@ -202,18 +161,54 @@ export default function BillingContainer(): JSX.Element {
...
 		[licensesData?.payload?.onTrial],
 	);
 
+	const { isLoading, data: usageData } = useQuery(
+		[REACT_QUERY_KEY.GET_BILLING_USAGE, user?.userId],
+		{
+			queryFn: () => getUsage(activeLicense?.key || ''),
+			onError: handleError,
+			enabled: activeLicense !== null,
+			onSuccess: processUsageData,
```
Comment (Rajat-Dabade) [inline frontend/src/container/BillingContainer/BillingContainer.tsx:186]: `useEffect` on line 212 can be remove as it is handle here.

File: frontend/src/utils/app.ts [hunk 2]
```diff
@@ -17,3 +17,16 @@ export const isCloudUser = (): boolean => {
 
 	return hostname?.endsWith('signoz.cloud');
 };
+
+export const checkVersionState = (
```
Comment (Rajat-Dabade) [inline frontend/src/utils/app.ts:21]:
```suggestion
export const checkVersionState = (
export const checkVersionState = (currentVersion: string, latestVersion: string): boolean => {
  const versionCore = currentVersion?.split('-')[0];
  return versionCore === latestVersion;
};

```

We can remove the unnecessary if statement as `split` function will return an array containing the original string as its single element if it doesn't find the specified separator in the string

**Review by Rajat-Dabade — CHANGES_REQUESTED** [review]
Some optimisation changes, but everything else looks good to me.

**Review by Rajat-Dabade — APPROVED** [review]
LGTM 🚀

**Pull request merged**